docs/source/asdf_in_fits.rst

```diff
@@ -0,0 +1,77 @@
+:py:mod:`stdatamodels.asdf_in_fits` contains functions to help migrate code that
```

Comment:
Add a title here, and a sphinx reference so that this page can be linked to externally.